msg/en_US/cubrid.msg

```diff
@@ -1347,7 +1347,7 @@ Check the path of the key file (_keys) and make sure that it includes proper key
 1276 Connected page server cannot be reached anymore.
 1277 Atomic replication error: %s.
 1278 Decompression failed (%s).
-1279 Error reserved for scalability development.
+1279 No page server connection is available. At least one connection is necessary.
```

Comment:
Shouldn't all the other language msg files be edited? That is how I know. But maybe you know better.

Reply:
Yes, you're right. They'll be edited after the review too. I should've mentioned it.